website/content/docs/connect/cluster-peering/create-manage-peering.mdx

```diff
@@ -0,0 +1,138 @@
+
+In “cluster-01,” run the ``generate-token`` command.
+
+```shell-session
+$ consul peering generate-token -peer="cluster-02"
+```
+
+The CLI outputs the peering token, which is an encoded string of alphanumeric characters and symbols.
+
+For the peering token to function correctly, you must enter the second cluster’s exact name in the ``peer`` parameter.
+
+## Establish a connection between clusters
+
+Next, use the peering token to establish a secure connection between the clusters. In the client agents of “cluster-02,” run the ``peering establish`` command. This command does not generate an output.
+
+```shell-session
+$ consul peering establish -peer="cluster-01" -token="eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJhZG1pbiIsImF1ZCI6IlNvbHIifQ.5T7L_L1MPfQ_5FjKGa1fTPqrzwK4bNSM812nW6oyjb8"
+```
+
+In the peer parameter, specify the first cluster’s exact name. The ``token`` parameter should include the entire peering token created in the first cluster.
+
+When you connect server agents through cluster peering, they will peer through their default partitions. To connect other partitions through server agents, you must specify the partitions you want to peer using the ``partition`` parameter.
+
+## Connect service endpoints
+
+After you establish a connection between the clusters, you need to create a configuration entry that defines the services that are available for other clusters. Consul uses this configuration entry to make services and support queries for service mesh available to peered clusters.
+
+First, create a configuration entry and specify the ``Kind`` as ``“exported-services”``.
+
+<CodeBlockConfig filename="peering-config.hcl">
+
+```shell-session
+Kind = "exported-services"
+Partition = "partition-name"
+
+Services = [
+  {
+    ## The name and namespace of the service to export.
+    Name      = "service-name"
+    Namespace = "default"
+
+    ## The list of peer clusters to export the service to.
+    Consumers = [
+      {
+        ## The peer name to reference in config is the one set
+        ## during the peering process.
+        Peer = "cluster-02"
+      }
+    ]
+```
+
+</CodeBlockConfig>
+
+Then, add the configuration entry to your cluster.
+
+```shell-session
+$ consul config write peering-config.hcl
+```
+
+Before you proceed, wait for the clusters to sync and make services available to their peers
+
+## Authorize connections from peers
+
+Before you can call services from peered clusters, you must set service intentions that authorize those clusters to use specific services. Consul prevents services from being exported to unauthorized clusters.
+
+First, create a configuration entry and specify the ``Kind`` as ``“service-intentions”``. Declare the service on "cluster-02" that can access the service in "cluster-01." The following example sets service intentions so that "backend-service" can access "frontend-service."
+
+<CodeBlockConfig filename="peering-intentions.hcl">
```

Comment:
same thing, if we don't expect users to run this. let's hide clipboard and highlight the relevant sections